tests/ws_integration.rs

```diff
@@ -176,8 +176,8 @@ mod ws_integration {
 			socket,
 			r#"
 			DEFINE SCOPE scope SESSION 24h
-				SIGNUP ( CREATE user SET email = $email, pass = crypto::argon2::generate($pass) )
```

Comment:
I'm actually testing the signin workflow here, that's why I didn't use the `ws_signin` method. Can we restore these test cases?